src/Common/constants.tsx

```diff
@@ -618,6 +618,16 @@ export const NOTIFICATION_EVENTS: NotificationEvent[] = [
     text: "Patient Note Added",
     icon: "l-notes",
   },
+  {
+    id: "PATIENT_PRESCRIPTION_CREATED",
+    text: "Patient Consultation Created",
+    icon: "l-capsule",
+  },
+  {
+    id: "PATIENT_PRESCRIPTION_UPDATED",
+    text: "Patient Consultation Updated",
+    icon: "l-capsule",
+  },
```

Comment:
The Id and text are not matching 🤔; Don't you think we should let users know that the prescription was updated insted of showing a generic message?

Reply:
Yup that is better will change it